location.go

```diff
@@ -0,0 +1,8 @@
+package thingfulx
+
+// Location is a data structure which holds an individual Thing's
+// geographic coordinates
+type Location struct {
+	Lng float64 `json:"longitude"`
+	Lat float64 `json:"latitude"`
```

Comment:
While looking at the metoffice example we've been working with, i noticed that the included an elevation as part of the location. Maybe we should consider adding that field for those data sources that happen to publish an elevation?